### DISCUSSION TITLE
nvidia driver 설치오류

### DISCUSSION THREAD

**Discussion by sun3702** [SKKUAutoLab/H-Mobility-Autonomous-Advanced-Course#102] [2024년도 교육 자료 관련 질문]
nvidia driver 설치과정 중 오류가 발생해서 github 코드설치가 중단되었습니다. driver 삭제 후 과정 이어서 진행하려했으나 삭제가 되지않습니다. 해결방법 문의 드립니다!

![image](https://github.com/user-attachments/assets/3558c2b6-5ed4-46aa-aa3d-1e3808f335f4)
![image](https://github.com/user-attachments/assets/185de6b6-f2bb-4aff-9066-d887ab6076c6)
![image](https://github.com/user-attachments/assets/554e0760-59e1-4a2d-ad4a-f3640eb5daef)
![image](https://github.com/user-attachments/assets/44877dff-ef9d-4f70-9bb1-def9b1d7f33b)

**Comment by Jinsun-Lee** [comment]
```
sudo rm /var/lib/apt/lists/lock
sudo rm /var/cache/apt/archives/lock
sudo rm /var/lib/dpkg/lock*
sudo dpkg --configure -a
sudo apt update
```
위의 명령어를 실행하시고 확인 바랍니다.  
이후 아래 명령어를 실행하시면 됩니다.  
```
cd ~/ros2_ws
./install.sh
```

**Reply by sun3702** [reply]
안녕하세요! sudo dpkg --configure -a 코드 실행시 첫번째 사진의 오류가 계속 발생합니다.

**Comment by HyeongKeunHong** [comment]
리눅스를 재설치하는게 빠를 것 같습니다.
윈도우로 부팅하신 뒤 하드디스크 파티션 설정창을 여시고, 우분투에 해당하는 볼륨을 삭제하신 뒤, 재설치하시면 됩니다.

재설치 후 영상자료를 따라가실 때에는 엔비디아 드라이버를 설치하는 부분은 생략하고 따라가셔도 무방합니다.
